askama_shared/src/generator.rs

```diff
@@ -1197,13 +1198,11 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
 
         self._visit_args(buf, args)?;
         buf.write(")?");
-        Ok(
-            if name == "safe" || name == "escape" || name == "e" || name == "json" {
-                DisplayWrap::Wrapped
-            } else {
-                DisplayWrap::Unwrapped
-            },
-        )
+        Ok(if FILTERS.contains(&name) {
+            DisplayWrap::Wrapped
+        } else {
+            DisplayWrap::Unwrapped
+        })
```

Comment:
I usually write these as a `match` these days, since they're more compact and even a little bit easier to read IMO. What do you think?

Reply:
So like this? Either is fine with me, so if you prefer with `match` then sure :)

```rust
Ok(match FILTERS.contains(&name) {
    true => DisplayWrap::Wrapped,
    false => DisplayWrap::Unwrapped,
})
```

Reply:
Yup!